subprojects/docs/src/samples/userguide/files/misc/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+// tag::mkdir-example[]
+task("ensureDirectory") {
+    doLast {
+        mkdir("images")
+    }
+}
+// end::mkdir-example[]
+
+// tag::move-example[]
+task("moveReports") {
+    doLast {
+        ant.withGroovyBuilder {
+            "move"(mapOf("file" to "${buildDir}/reports", "todir" to "${buildDir}/toArchive"))
```

Comment:
:x: can be simplified to `"move"("file" to "$buildDir/reports", "todir" to "$buildDir/toArchive")`